common-core-open/src/main/java/edu/isi/nlp/files/KeyValueSources.java

```diff
@@ -92,4 +96,35 @@ private KeyValueSources() {
     }
     return new ZipKeyValueSource(zipFile, ret.build());
   }
+
+  /**
+   * Creates a new source using a zip file that originated from a character or byte sink in the
+   * Python module vistautils.key_value, using default parameters. Maps each entry in the zip file
+   * to a unique key that is specified by a file in the zip named "__keys". (see
+   * https://github.com/isi-vista/vistautils/blob/master/vistautils/key_value.py#L95). The caller
```

Comment:
* give the link to the root of the `vistautils` GitHub repository, but otherwise refer to parts of the code by module, class, and method names because the line numbers will change over time.